website/docs/r/datadog_sso.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+subcategory: "Datadog"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_datadog_monitor_sso"
+description: |-
+  Manages SingleSignOn on the datadog Monitor.
+---
+
+# azurerm_datadog_monitor_sso
+
+Manages SingleSignOn on the datadog Monitor.
+
+## Example Usage
+
+### Enabling SSO on monitor
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-datadog"
+  location = "West US 2"
+}
+resource "azurerm_datadog_monitor_sso" "test" {
+    name = "example-monitor"
+    resource_group_name = azurerm_resource_group.example.name
+    singlesignon_state = "Enable"
+    enterpriseapp_id = "XXXX"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this datadog Monitor.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the datadog Monitor should exist.
+
+* `singlesignon_state` - (Required) The state of SingleSignOn configuration.
+
+* `enterpriseapp_id` - (Required) The application Id to perform SSO operation.
+
+--- 
+
+* `configuration_name` - (Optional) The name of the SingleSignOn configuration.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported:
+
+* `id` - The ID of the Tagrules on datadog monitor.
+
+* `type` - The type of the monitor resource.
+
+* `provisioning_state` - The state of Datadog monitor.
```

Comment:
We can remove this property.

Reply:
These properties we show to the users.

Reply:
Typically, we don't expose this property (as you can see from other resources).

Reply:
Removed